encoding/ccf/simpletype.go

```diff
@@ -140,6 +140,15 @@ const ( // Cadence simple type IDs
 	SimpleTypeHashableStruct
 	SimpleTypeFixedSizeUnsignedInteger
 
+	// Fix* types
+
+	SimpleTypeFix128
```

Comment:
As a follow-up task, also the spec should be updated to reflect the addition: https://github.com/onflow/ccf/blob/main/ccf_specs.md